### PR TITLE
prow/override: Trim whitespace in /override  <context>

### PR DESCRIPTION
**Pull request by cgwalters**
It's really confusing if one types e.g.
`/override  ci/prow/blah`
because Markdown compresses two spaces into one when rendered,
so the error is not obvious.  Let's be nice to the humans
and trim whitespace.

Signed-off-by: Colin Walters <walters@verbum.org>

### CODE REVIEW

File: prow/plugins/override/override.go [hunk 1]
```diff
@@ -286,7 +286,7 @@ func handle(oc overrideClient, log *logrus.Entry, e *github.GenericCommentEvent,
 			log.Debug(resp)
 			return oc.CreateComment(org, repo, number, plugins.FormatResponseRaw(e.Body, e.HTMLURL, user, resp))
 		}
-		overrides.Insert(m[2])
+		overrides.Insert(strings.TrimSpace(m[2]))
```
Comment (alvaroaleman) [inline prow/plugins/override/override.go:289]: @cgwalters please add a tetscase for this
/ok-to-test

Comment (cgwalters) [reply prow/plugins/override/override.go:289]: Done!

**Comment by cgwalters** [comment]
I've hit this a number of times, here's one example https://github.com/ostreedev/ostree/pull/2189#issuecomment-686681560 - you can see the issue by viewing the raw markdown.

**Review by stevekuznetsov — APPROVED** [review]

**Comment by cgwalters** [comment]
Hah, fixed the extra whitespace in my test for extra whitespace...

**Review by alvaroaleman — APPROVED** [review]

**Comment by alvaroaleman** [comment]
failure looks transient
/retest
